src/main/scala/org/ergoplatform/http/api/SortDirection.scala

```diff
@@ -0,0 +1,14 @@
+package org.ergoplatform.http.api
+
+/**
+  * Encoded results sorting direction (in ascending or descending order)
+  */
+object SortDirection {
+
+  type Direction = Byte
+
+  val ASC: Direction = 1.asInstanceOf[Direction]
+  val DESC: Direction = 0.asInstanceOf[Direction]
+  val INVALID: Direction = (-1).asInstanceOf[Direction]
```

Comment:
@ccellado, maybe .toByte would be more clean, don't you think?

Reply:
removed asInstanceOf in https://github.com/ergoplatform/ergo/pull/2067/commits/e93daf02c121e2a6db13363f02480caa66da126f